src/content/zh-TW/2020/performance.md

```diff
@@ -0,0 +1,387 @@
+---
+part_number: II
+chapter_number: 9
+title: 效能
+description: 2020 Web Almanac 的效能章節涵蓋了 Core WebVitals、Lighthouse Performance Score、First Contentful Paint (FCP) 以及 Time to First Byte (TTFB)。
+authors: [thefoxis]
+reviewers: [borisschapira, rviscomi, obto, noamr, Zizzamia, exterkamp]
+analysts: [max-ostapenko, dooman87]
+translators: [cybai, aszx87410, ArvinH, andy0130tw, AbbyTsai]
+thefoxis_bio: Karolina 是 <a href="https://calibreapp.com/">Calibre</a> 的產品設計主管。她主要負責打造最全面的速度監測平台。她同時也是為您蒐集效能相關訊息與資訊的 <a lang="en" href="https://perf.email/">Performance Newsletter</a> 發起人之一。Karolina 也<a href="https://calibreapp.com/blog/category/web-platform">時常寫有關於效能如何影響使用者體驗的文章</a>。
```

Comment:
Awesome! Will update all the links which should be highlighted! Thanks!

Reply:
Nice spotlight. a thought come into my mind as the word might appear many time in chapter. Would we think it is good to highlight all of them in language? or leave them except highlighting the first word. @HTTPArchive/traditional-chinese-translators @bazzadp  would love to hear your idea. thanks.

Reply:
As curiosity, what's a deep reason to declare another language attribute in Chinese, for example: ` <span lang="en">this is English</sapn>`, while it may be shown as string as well. is it? feel free to share your thoughts. thanks.

Reply:
> a thought come into my mind as the word might appear many time in chapter. Would we think it is good to highlight all of them in language? or leave them except highlighting the first word.

We only highlight links in this way, not English words. It wouldn’t be typical to have multiple links to same place and even if they are used, then yes they ideally all should be highlighted as in a different language from the original article so reader knows before they waste time clicking.

> As curiosity, what's a deep reason to declare another language attribute in Chinese, for example: <span lang="en">this is English</sapn>, while it may be shown as string as well. is it? feel free to share your thoughts. thanks.

This is used to indicate to screen-readers to indicate the language should be read differently (see [WCAG 3.1.2](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts). It is also [possible to use `<i lang="en">` instead of `<span lang="en">`](https://stackoverflow.com/questions/30322839/which-element-should-i-use-for-short-group-of-foreign-langague-words) to show this in italics. Less useful for Chinese where the fact it is in English is more obvious, but can be handy for other Latin languages (e.g. French) to highlight the phrase more as a foreign language phrase.

We strive to have the very best in Accessibility on this site so do recommend translators take these extra steps where at all possible.

Reply:
Fixed in 7c937c4